### ISSUE TITLE
[Feature Request] Horizontally Scrollable Code blocks in Preview Mode

### ISSUE THREAD

**Issue by dvdmtw98** [AnubisNekhet/AnuPpuccin#86]
### Feature Description

The ability to scroll code blocks when in Preview Mode. Scrolling would be extremely useful for reading code blocks that have very long (horizontal) lines.

Additionally, setting code block to `No Wrap` in Preview Mode currently does not seem to have any effect

![Current Code block](https://user-images.githubusercontent.com/31100937/219922353-c98a5eb5-ad59-45bc-ab8e-662bded52295.png)

### Additional context

Image from Things Theme which has this feature

![Scrolling Code Block](https://user-images.githubusercontent.com/31100937/219922272-11317b82-f34c-415b-9789-adbbc79337b2.png)

Sample Code block to testing

```csharp
// Named Arguments 
PrintOrderDetails(productName: "Red Mug", sellerName: "Gift Shop", orderNum: 31);

// Optional Arguments 
public void ExampleMethod(int required, string optionalstr = "default string", int optionalint = 10) {}
```

**Comment by AnubisNekhet** [comment]
Codeblock scrolling does not apply to blocks with syntax highlighting, though I'll add an option to toggle that as well.

**Comment by dvdmtw98** [comment]
That would be great. 

I tried removing the syntax highlighting from the above code block and scrollbar does show up but it also turns the entire code block into a single line of text. Not sure if this is a bug but this is definitely not ideal.

![Code block with Wrap](https://user-images.githubusercontent.com/31100937/219953571-edf805ad-1b26-422d-be59-568c018b61c6.png)

**Comment by AnubisNekhet** [comment]
Yep, I've noticed that too. I've changed the wrapping from `nowrap` to `pre` which preserves newlines.

**Comment by AnubisNekhet** [comment]
Implemented with v1.4.0!
